static/app.template.html

```diff
@@ -123,6 +127,10 @@
   });
 });
 
+$(function() {
+     $("#datePicker").datepicker({ dateFormat: 'yy-mm-dd'}); 
```

Comment:
date format should be pt-BR to show (dd/mm/yyyy).

Also, wouldn't be better to have a css called datepicker? In that way, input object we add this css would be a date bicker. Is that correct?